xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/documentTags.vm

```diff
@@ -109,9 +109,12 @@
 ## Note that the form_token parameter needs to be kept before the xredirect parameter since the JS code might replace the latter
 ## All that would need a cleaner fix in the javascript of tags.
 <form action="$doc.getURL('view', "xpage=documentTags&amp;xaction=add&amp;form_token=$!{escapetool.url($services.csrf.token)}&amp;xredirect=${xredirect}")" method="post" class="tag-add-form">
-  <div>
+  <div id="tag-add-form-suggest">
     <label for="tag">$services.localization.render('core.tags.add.label')<br/>
-      <input class="input-tag" type="text" id="tag" name="tag" autocomplete="off"/></label><br/>
+      <input class="input-tag" type="text" id="tag" name="tag" autocomplete="off"/>
+    </label>
```

Comment:
Is the removal of the trailing `<br/>` intentional?

Reply:
Yes :)
It wasn't needed with the other changes to style. Semantically it did not hold much importance.

A source about the semantics of line breaks, and its conclusion:
https://www.joedolson.com/2007/01/is-a-br-tag-semantic/

> I don’t think that a line break has semantic value. However, there are situations where a line break is required because custom dictates it. Other methods of formatting may have their value, but the use of an occasional line break isn’t going to make or break a site’s quality code.